tests/testthat/test-expect_s3_class_linter.R

```diff
@@ -0,0 +1,97 @@
+test_that("expect_s3_class_linter skips allowed usages", {
+  # expect_s3_class doesn't have an inverted version
+  expect_lint("expect_true(!inherits(x, 'class'))", NULL, expect_s3_class_linter())
+  # NB: also applies to tinytest, but it's sufficient to test testthat
+  expect_lint("testthat::expect_s3_class(!inherits(x, 'class'))", NULL, expect_s3_class_linter())
```

Comment:
Is the expextation function intentional? Looking for `expect_true` here.

Reply:
nice! fixed